### PR TITLE
Add eslint rules

### PR DESCRIPTION
**Pull request by Aschen**
## Description

Add following eslint rules to comply with standards:
 - [comma-dangle](https://eslint.org/docs/rules/comma-dangle) rule (only for multiline)
 - [object-shorthand](https://eslint.org/docs/rules/object-shorthand)

### CODE REVIEW

File: test/core/network/accessLogger.test.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('#AccessLogger', () => {
     };
 
     mockRequire('worker_threads', {
-      Worker: function (...args) {
+      Worker (...args) {
```
Comment (OlivierCavadenti) [inline test/core/network/accessLogger.test.js:31]: Worker is not a constructor (miss '=>' ?)

**Comment by OlivierCavadenti** [comment]
still needed since we have prettier ? @Aschen

**Comment by Aschen** [comment]
Yes because this PR contain the rules that fit with Prettier formatting standards but I need to fix the plugin first

**Comment by Aschen** [comment]
We use Prettier now

**Pull request closed without merging**